drivers/include/tmp006.h

```diff
@@ -84,8 +84,8 @@ extern "C"
 {
 #endif
 
-#ifndef TMP006_I2C_ADDRESS
-#define TMP006_I2C_ADDRESS         0x41 /**< TMP006 Sensor Default Address */
+#ifndef TMP006_ADDR
+#define TMP006_ADDR                0x40 /**< TMP006 Sensor Default Address */
```

Comment:
Why is the default state of the pin C1 and B1 low ? or why you changed the address?